src/coreclr/jit/lsrabuild.cpp

```diff
@@ -885,9 +885,7 @@ regMaskTP LinearScan::getKillSetForCall(GenTreeCall* call)
 #ifdef SWIFT_SUPPORT
     // Swift calls that throw may trash the callee-saved error register,
     // so don't use the register post-call until it is consumed by SwiftError.
-    // GTF_CALL_M_SWIFT_ERROR_HANDLING indicates the call has a SwiftError* argument,
-    // so the error register value will eventually be consumed post-call.
-    if ((call->gtCallMoreFlags & GTF_CALL_M_SWIFT_ERROR_HANDLING) != 0)
+    if (call->HasSwiftErrorHandling())
```

Comment:
If this ends up being expensive TP wise you can add some handling in `AddedWellKnownArg` to make it cheaper.

Reply:
I guess it'll be cheap with the check for the calling convention in `HasSwiftErrorHandling`

Reply:
Yeah, I don't think this will be too expensive in the general case, though if we want to speed up the Swift error handling case, I can add some new state to `CallArgs` to avoid the lookup.

Reply:
Actually, updating this state in `AddedWellKnownArg` would probably affect TP in the general case, so maybe it's best to leave the unusual case slower than it could be, if it means the general case is untouched.